lib/node_modules/@stdlib/lapack/base/dlange/lib/isoperation.js

```diff
@@ -0,0 +1,62 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var contains = require( '@stdlib/array/base/assert/contains' ).factory;
+
+
+// VARIABLES //
+
+var NORMS = require( './norms.json' ).norms;
+
+
+// MAIN //
+
+/**
+* Tests whether an input value is a supported operation.
+*
+* @name isOperation
```

Comment:
```suggestion
* @private
* @name isOperation
```

This function should be considered a private internal function relative to the package. It is not for public consumption.